### PR TITLE
Report/user activities

### PR DESCRIPTION
**Pull request by AugustBugge**

### CODE REVIEW

File: docs/diagrams/AuthorizedUserActivities.drawio.svg
Comment (Oliverst8) [inline docs/diagrams/AuthorizedUserActivities.drawio.svg:1]: Is public timeline the only option I have when I am on the logout page

Comment (Oliverst8) [inline docs/diagrams/AuthorizedUserActivities.drawio.svg:1]: When recheeping something you also get redirected to your private timeline

Comment (Oliverst8) [inline docs/diagrams/AuthorizedUserActivities.drawio.svg:1]: The arrows pointing away from public timeline to the left side of the picture, it looks like I have to clikc other users displayname to be able to recheep or follow

Comment (Oliverst8) [inline docs/diagrams/AuthorizedUserActivities.drawio.svg:1]: It is also possible for me to cheep fro my private timeline

File: docs/diagrams/UnauthorizedUserActivities.drawio.svg
Comment (Oliverst8) [inline docs/diagrams/UnauthorizedUserActivities.drawio.svg:1]: This diagram makes me belive I can do things like cheep, even tho I am not logged in. I also do not have the options I have in the other diagram, so It like its half finished

Comment (Oliverst8) [inline docs/diagrams/UnauthorizedUserActivities.drawio.svg:1]: It is also possible to change pages

Comment (Oliverst8) [reply docs/diagrams/UnauthorizedUserActivities.drawio.svg:1]: This is for both diagrams

File: docs/report.md [hunk 1]
```diff
@@ -91,10 +91,84 @@ Illustrate the server component and to where it is deployed, illustrate a client
 
 ### User activities
 
-Illustrate typical scenarios of a user journey through your _Chirp!_ application.
-That is, start illustrating the first page that is presented to a non-authorized user, illustrate what a non-authorized user can do with your _Chirp!_ application, and finally illustrate what a user can do after authentication.
+Unregistered users start on the public timeline and can either register or login to become a authorized user. 
+They can also view the cheeps on the public timeline, change page, and view other users private timeline, by clicking on their names.
+Once authorized you can do the same as an unauthorized user, but in a addition they can write new cheeps, follow other users, or recheep their cheeps.
```
Comment (Oliverst8) [inline docs/report.md:96]: You cant create a new account

**Review by iiinaD — APPROVED** [review]
Looks good

**Pull request merged**